liquibase-core/src/main/java/liquibase/diff/output/changelog/core/UnexpectedTableChangeGenerator.java

```diff
@@ -47,7 +47,7 @@ public Change[] fixUnexpected(DatabaseObject unexpectedObject, DiffOutputControl
 
         for (Column column : unexpectedTable.getColumns()) {
             control.setAlreadyHandledUnexpected(column);
-        };
+        }
```

Comment:
**Today I Learned** ... Java for loops don't require a closing `;`.  @arturobernalg, I'm learning a lot of interesting Java nuances with your code cleanup PRs!

Reply:
Lol ... also, please tell me if I learned the wrong thing! I'm not a Java dev by trade.